composer.lock

```diff
@@ -805,6 +805,138 @@
+                "type": "git",
+                "url": "https://github.com/drunomics/contentpool-replication.git",
+                "reference": "09ee395bfe7cd2955c9b2cb68cf200453dc179e2"
+            },
+            "dist": {
+                "type": "zip",
+                "url": "https://api.github.com/repos/drunomics/contentpool-replication/zipball/09ee395bfe7cd2955c9b2cb68cf200453dc179e2",
+                "reference": "09ee395bfe7cd2955c9b2cb68cf200453dc179e2",
+                "shasum": ""
+            },
+            "require": {
+                "drupal/replication": "^1.0@beta"
+            },
+            "type": "drupal-module",
+            "extra": {
+                "branch-alias": {
+                    "dev-8.x-1.x": "1.0.x-dev"
+                },
+                "patches": {
+                    "drupal/replication": {
+                        "#2989572: Refactor Changes class to be more modular": "https://www.drupal.org/files/issues/2018-07-31/refactor-changes-2989572-2.patch"
+                    }
+                }
+            },
+            "notification-url": "https://packagist.org/downloads/",
+            "license": [
+                "GPL-2.0-or-later"
+            ],
+            "authors": [
+                {
+                    "name": "drunomics GmbH",
+                    "email": "hello@drunomics.com"
+                }
+            ],
+            "description": "Replication filter for contentpool entities.",
+            "time": "2018-08-22T07:05:08+00:00"
+        },
+        {
+            "name": "drunomics/service-utils",
+            "version": "5.x-dev",
+            "source": {
+                "type": "git",
+                "url": "https://github.com/drunomics/service-utils.git",
+                "reference": "edc4365563144a81049bfc99388467a73fb6bc8f"
+            },
+            "dist": {
+                "type": "zip",
+                "url": "https://api.github.com/repos/drunomics/service-utils/zipball/edc4365563144a81049bfc99388467a73fb6bc8f",
+                "reference": "edc4365563144a81049bfc99388467a73fb6bc8f",
+                "shasum": ""
+            },
+            "require": {
+                "drupal/core": "8.5.*"
+            },
+            "require-dev": {
+                "drupal/coder": "^8.2",
+                "phpunit/phpunit": "~4.8"
+            },
+            "type": "library",
+            "autoload": {
+                "psr-4": {
+                    "drunomics\\ServiceUtils\\": "src/"
+                }
+            },
+            "notification-url": "https://packagist.org/downloads/",
+            "license": [
+                "GPL-2.0+"
+            ],
+            "authors": [
+                {
+                    "name": "drunomics GmbH",
+                    "email": "hello@drunomics.com"
+                }
+            ],
+            "description": "Provides setter injection traits in order to ease dependency injection of services.",
```

Comment:
Actually, this file is dumb as it won't be used anywhere. I think it was there in the past for installing phpcs for checking phpcs, but that's now done at the project level also. Thus, let's remove the lock file here to avoid confusion.